alf/environments/suite_carla.py

```diff
@@ -569,6 +590,16 @@ def get_current_time_step(self, current_frame):
                 self._unrecorded_distance_reward = 0
         reward += distance_reward
 
+        red_light_id = self._alf_world.is_running_red_light(self._actor)
+        if red_light_id is not None and red_light_id != self._prev_violated_red_light_id:
+            logging.info("actor=%d frame=%d RED_LIGHT" % (self._actor.id,
+                                                          current_frame))
+            reward_vector[Player.REWARD_RED_LIGHT] = 1.
```

Comment:
Why assign 1 here?

Reply:
Currently, this is just for indicating the event. So it does not matter. And it is different from the actual reward for running red light

Reply:
From my understanding, if we learn with multiple rewards, then there are multiple values learned. So basically the learned value of collision or red light is never used (except display)?

Reply:
Right, it's not used. Though it can help to shape the representation.